test/hotspot/jtreg/runtime/cds/appcds/test-classes/DumpClassListApp.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2016, 2019, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Is this a new file?

Reply:
Yes, it is a new file. In my first commit I included 2016 in the copyright by mistake, so I removed it in the second commit.